ejb-txn-remote-call/server/pom.xml

```diff
@@ -207,70 +209,38 @@
+                                </feature-pack>
+                                <feature-pack>
+                                    <location>org.wildfly.cloud:wildfly-cloud-galleon-pack:${version.pack.cloud}</location>
+                                </feature-pack>
+                                <feature-pack>
+                                    <location>org.wildfly:wildfly-datasources-galleon-pack:${version.pack.datasources}</location>
+                                </feature-pack>
+                            </feature-packs>
+                            <discover-provisioning-info>
+                                <version>${version.server}</version>
+                                <add-ons>
+                                    <add-on>postgresql</add-on>
+                                    <add-on>wildfly-cli</add-on>
+                                </add-ons>
+                                <context>cloud</context>
+                                <!-- Some JNDI lookup to ignore -->
+                                <failsOnError>false</failsOnError>
+                                <profile>ha</profile>
+                            </discover-provisioning-info>
+                            <extra-server-content-dirs>
+                                <extra-content>extra-content</extra-content>
+                            </extra-server-content-dirs>
+                            <build>false</build>
                         </configuration>
                         <executions>
                             <execution>
-                                <id>package</id>
-                                <phase>package</phase>
                                 <goals>
                                     <goal>package</goal>
                                 </goals>
-                                <configuration>
-                                    <recordProvisioningState>true</recordProvisioningState>
-                                    <discover-provisioning-info>
-                                        <version>${version.server}</version>
-                                        <add-ons>
-                                            <add-on>postgresql</add-on>
-                                            <add-on>wildfly-cli</add-on>
-                                        </add-ons>
-                                        <!-- Some JNDI lookup to ignore -->
-                                        <failsOnError>false</failsOnError>
-                                        <profile>ha</profile>
-                                    </discover-provisioning-info>
-                                </configuration>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
-            <id>openshift</id>
```

Comment:
we no longer have openshift profile in server module, is this expected?

Reply:
Yep, I realised we can just trust the `provisioned-server` profile